link_drag0n.js

```diff
@@ -0,0 +1,76 @@
+
+const fs = require('fs');
+let { Agent } = require('./agents/AddressableAgent.js');
+
+const subprocess = require('child_process');
+
+let agent = new Agent("LinkDragonAgent", "hi");
+agent.connect();
+agent.login();
+
+let allowedUsers = ['gene', 'liz', 'roan'];
+
+const linkCastleFile = 'linkCastle.json';
+
+function readLinkFile() {
+    try {
+        return JSON.parse(fs.readFileSync(linkCastleFile, 'utf8'));
+    } catch (error) {
+        console.error("Error reading the link file:", error);
+        return {};
+    }
+}
+
+function writeLinkFile(data) {
+    try {
+        fs.writeFileSync(linkCastleFile, JSON.stringify(data, null, 2), 'utf8');
+    } catch (error) {
+        console.error("Error writing to the link file:", error);
+    }
+}
+
+function storeLink(link, user) {
+    const data = readLinkFile();
```

Comment:
after adding a database, have the agent send a message to a new channel, linkdragondata, where another agent goes and fetches the link